docs/config.toml

```diff
@@ -9,8 +9,8 @@ theme= "iceberg-theme"
   siteType = "docs"
   search = true
   versions.iceberg = "" # This is populated by the github deploy workflow and is equal to the branch name
-  versions.nessie = "0.59.0"
-  latestVersions.iceberg = "1.3.0"  # This is used for the version badge on the "latest" site version
+  versions.nessie = "0.65.0"
```

Comment:
I think this should stay as it was before, because 1.3.1 uses https://github.com/apache/iceberg/blob/apache-iceberg-1.3.1/versions.props#L25